src/output.ts

```diff
@@ -1,7 +1,15 @@
 import * as readline from 'readline';
 import chalk from 'chalk';
+import highlight from 'cli-highlight';
+import {
+  codeBlockAnnotationsRegExp,
+  langRegExp,
+  splitCodeBlocks,
+  stripCodeblockAnnotations,
+} from './syntax-highlighting';
 
 let verbose = false;
+let syntaxHighlighting = true;
```

Comment:
Perhaps we could automatically detect if the output is a terminal and supports color. I will make a quick PR for that, as we also output ai message colors which we should turn off.

Reply:
Use `chalk.supportsColor` to detect output highlighting. Check if it's not working already, as Chalk automatically picks up `--colors`/`--no-colors` CLI options 🤯